### ISSUE TITLE
Add Stripe Payments Order model to the docs

### ISSUE THREAD

**Issue by andrelopez** [enupal/docs#6]

**Comment by lucashfreitas** [comment]
Hello Andre,

I am currently maintaining a project that is using the plugin and after an update the part of the code that gets the order return URL is not working. I also didn't find any docs of what options/how the order object/methods looks like.

This is the is the part of the code I am using for the `customer` and `admin` email notification:


`    <title>{{ 'Order: '|t }} {{ order.number }}</title> //what is order? what are the available fields?`

This part of the code is not working after the update to the latest version:
```
<a style="text-decoration: none; color: #ffffff;" href="{{ order.getPaymentForm().getReturnUrl(order) }}">Order info</a> 
```

Trying to figure out why `order.getPaymentForm().getReturnUrl(order)` is returning an empty string after an update to the latest plugin version.

Please let me know if you need any additional information,

Thank you very much!

**Comment by andrelopez** [comment]
Hi @lucashfreitas 

We just released Stripe Payments 3.1.4 with a fix to that issue
